dask/dataframe/core.py

```diff
@@ -810,6 +810,34 @@ def min(self, axis=None, skipna=True):
                                  skipna=skipna, axis=axis,
                                  meta=meta)
 
+    def idxmaxmin(self, fn, axis=None, skipna=True):
+        if fn == 'idxmax':
+            idxfn = _idxmax
+        elif fn == 'idxmin':
+            idxfn = _idxmin
+        else:
+            raise ValueError('Function must be idxmax or idxmin')
+
+        axis = self._validate_axis(axis)
+        # if axis == 1:
+            # return map_partitions(idxfn, None, self,
+                                  # token=self._token_prefix + fn,
+                                  # skipna=skipna, axis=axis)
+        # else:
```

Comment:
Dask.dataframe only partitions itself vertically.  All columns of a particular row are in the same partition.  See http://dask.pydata.org/en/latest/dataframe-partitions.html